.gitlab-ci.yml

```diff
@@ -8,40 +8,55 @@ default:
   retry:
     max: 2
 
-.docker-env:                       &docker-env
-  image:                           "paritytech/ci-linux:staging"
+.docker-env:
+  image:                           "paritytech/ci-unified:bullseye-1.71.0"
```

Comment:
What are the effects of this and why was this changed/needed?

Reply:
We replaced `ci-linux` with `ci-unified`. It has transparent tags naming, flexible choice of 
Rust versions, and so on. More - https://hub.docker.com/repository/docker/paritytech/ci-unified

Reply:
We are switching to our new `ci-unified` image. We decided to have "one for all" image and finally added versioning to it.